agents/heuristics_resource/fence_heuristics_resource.py

```diff
@@ -0,0 +1,160 @@
+#!/usr/libexec/platform-python -tt
```

Comment:
This should be updated to `#!@PYTHON@ -tt` to make it use the auto-detected/user-specified Python binary from running ./configure.

Reply:
Done.